### ISSUE TITLE
How can I configure my certificates so my JWKS is valid for external services?

### ISSUE THREAD

**Issue by gcarrarom** [goauthentik/authentik#4701]
## Question 

I am trying to setup an Akeyless Auth Method of the Oauth2 kind. The setup - as expected - requires me to supply a JWKS endpoint for the validation of the tokens. I have tried supplying multiple Certificates, including Let's Encrypt signed certificates on the Provider for AKeyless that I've created in Authentik, but it always fails on the verification of the endpoint.
How can I ensure that the JWKS endpoint responds with valid information?

![CleanShot 2023-02-15 at 15 47 57](https://user-images.githubusercontent.com/10549675/219153562-3a83ddfc-add9-42bb-bffe-fb6aa3b4b6fc.png)

**Version and Deployment (please complete the following information):**
 - authentik version: 2023.2.1
 - Deployment: helm

Here's my JWKS endpoint: https://authentik.fancywhale.ca/application/o/akeyless/jwks/

It would be great to get some direction on what might be happening. How can I validate the validity of the JWKS endpoint?

**Comment by BeryJu** [comment]
The JWKS response looks valid to me, and I couldn't find anything with the error you posted above, so I'm not sure what the issue is. JWKS Certificates don't have to be signed so that shouldn't cause any issues either. Are there any logs of Akeyless? Can you post the full screenshot of your configuration?

**Comment by gcarrarom** [comment]
I can't find any logs to help me out, but here's the command I've used to create it:
```bash
akeyless create-auth-method-oauth2 --name authentik-oauth --jwks-uri https://authentik.fancywhale.ca/application/o/akeyless/jwks/ --unique-identifier user_login
```

I've contacted the support of Akeyless. Seems to be a security concern for them. No more description given. I've tried with my self-hosted gitlab JWKS that works on other integrations, and it also failed. I believe this to be a validation process they must do on the certificates of JWKS endpoints. Could it be only whitelisted certificates? Nevertheless, seems like something we can't fix with Authentik, eh?
